src/Arrow/index.tsx

```diff
@@ -13,10 +13,11 @@ const transformMap = {
 	left: 'translateX(-50%) rotate(45deg)',
 };
 
-function useArrowStyles(primaryPlacement, arrowSize) {
+function useArrowStyles(primaryPlacement: string, arrowSize: Number) {
 	const arrowStyles = useMemo(
 		() => ({
-			position: 'absolute',
+			// casting field to itself to enforce type as valid
+			position: 'absolute' as 'absolute',
```

Comment:
Why's this necessary? Is there something special about the position property or the absolute value that necessitates this? If so, that should be added to the comment.

Reply:
Yes. It doesn't recognise the unmodified string as a valid option defined on the position interface, it recognises it simply as a string, but casting it to itself fixes this. I am not well enough versed on TS yet to explain why.